libs/ruta.js

```diff
@@ -0,0 +1,9 @@
+
+import path from 'path';
+
+// let rute;
+export const absolutePath=(pathFromCli)=>{
+  let rute='';
+  return path.isAbsolute(pathFromCli) === true ? rute = pathFromCli : rute=path.resolve(pathFromCli);
```

Comment:
```suggestion
  return path.isAbsolute(pathFromCli) ? pathFromCli : path.resolve(pathFromCli);
```

Te recomiendo leer más sobre el operador ternario. Recuerda que maneja 3 expresiones. Por otro lado, recuerda que la función `path.isAbsolute` retorna un valor booleno, por lo que no hay necesidad de compararlo con `true` para al final retornar el resultado de esa comparación. Retornemos directo lo que retorna `isAbsolute`